frontend/src/store/facilityBooking/action.ts

```diff
@@ -380,25 +384,49 @@ export const resetBooking = () => (dispatch: Dispatch<ActionTypes>) => {
 }
 
 export const handleCreateNewBooking = (
-  facilityID: number,
-  eventName: string,
+  facilityID: number | undefined,
+  eventName: string | undefined,
   startTime: number | null,
   endTime: number | null,
-  endDate?: number | null,
+  endDate?: number,
   ccaID?: number,
   description?: string,
+  forceBook?: boolean,
 ) => async (dispatch: Dispatch<ActionTypes>) => {
+  // const requestBody = {
+  //   facilityID: facilityID,
+  //   eventName: eventName,
+  //   userID: localStorage.getItem('userID'),
+  //   ccaID: ccaID,
+  //   startTime: startTime,
+  //   endTime: endTime,
+  //   endDate: endDate,
+  //   description: description,
+  //   forceBook: forceBook,
+  // }
```

Comment:
remove comments!

Reply:
this should be uncommented, remove the next const

Reply:
Resolved in commit ca5730df3da1e495b5afa9c949a610730cb97605.